Akismet/AkismetReal.php

```diff
@@ -83,6 +83,25 @@ public function isSpam(array $data)
         }
     }
 
+    function submitSpam(array $data)
+    {
+        $fullData = array_merge($this->getRequestData(), $data);
+
+        if ($this->throwExceptions) {
+            return $this->adapter->submitSpam($fullData);
```

Comment:
Does that mean you can't or won't change this? I'd prefer it cleaned up, but I understand if you have a reason to keep it this way.